content/docs/guides/price-schedules.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: 'Same Product, Multiple Price Schedules'
+section: 'Guides'
+priority: 6
+hidden: false
+---
+
+import { Link } from 'gatsby'
+
+Price schedules are one of the critical factors related to <Link to="main-concepts/product-visibility">product visibility</Link> and there are several ways that you can build relationships between products and price schedules.  One of the scenarios you may wish to achieve is one product with multiple price schedules.  _Why_ would you possibly want one product that had multiple different prices based on buyer/user?  Lets take a look at the example scenario below, and walk through a solution to achieve the desired result.
+
+> As a seller company, I sell two different memberships to my b2b ecommerce site.  The first is **Startup** and the second is **Enterprise**.  The **Enterprise** membership is considerably more expensive than the **Startup** membership, therefore I want to provide a slightly lower product pricing to my **Enterprise** customers.  Specifically, Jane Doe from Best Buy has signed up for an **Enterprise** membership, and John Deer from TechWorld has signed up for a **Startup** membership.
+
+How would we go about achieving this effect you might ask?  After reviewing the considerations, let's dive in!
+
+## Considerations
+
+1. In our example, we will be assigning price schedules to products and parties at the Buyer level, however you can get even more granular than that, and assign Product/Price Schedule combinations at the User Group level within a single Buyer.  This allows you further flexibility within your product visibility.
+
+2. This guide assumes that all products, categories and catalogs have been created successfully, with proper assignments.  Make sure that your product is assigned to a catalog that the buyer's users have access to, or you will have issues when trying to create your product assignments.
+
+
+## Create a Price Schedule
+
+First, lets create the price schedule that is to be associated with the Enterprise membership.  For the length of this example, we will use "USB Cord" as our Product.  
+
+Lets assume there is no tax or shipping costs, and the minimum order quantity is 1.  Our call to the OrderCloud API should like this (If successful, our call should return a `201 Created` response):
+
+```
+Method
+POST
+
+URL
+https://api.OrderCloud/v1/priceschedules
+
+Data
+{
+  "ID": "enterprise-priceschedule-id"
+  "Name": "USB Cord",
+  "MinQuantity": 1,
+  "PriceBreaks": [
+    {
+      "Quantity": 1,
+      "Price": 3.99
+    }
+  ]
+}
+
+Unflatten
+yes
+
+Headers
+application/json
+Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9...
+```
+
+Next, lets make the exact same call, but change the `PriceBreaks` field to adjust for the **Startup** pricing.  The `PriceBreaks` array in your second call should look like this: 
+
+```json
+{
+  ...
+  "PriceBreaks": [
+    {
+      "Quantity": 1,
+      "Price": 5.99
+    }
+  ]
+}
+```
```

Comment:
lol i was about to say this might be a good opportunity to use the highlighting feature: 
![image](https://user-images.githubusercontent.com/16483662/65719813-3615d100-e06c-11e9-9f65-40ea52a14305.png) but it's broken :( Looks like a css thing because i can see `gatsby-highlight-code-line` class is applied but the background color doesn't seem to be different than the other lines

Reply:
Wouldn't a user also want to change the PriceSchedule ID so that its `"ID": "startup-priceschedule-id"`